src/pages/app-development/starter-kit/customers.md

```diff
@@ -10,4 +10,39 @@ keywords:
  - Tools
 ---
 
-# Customers data flow
+# `customer` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
+## Create
+
+Adobe Commerce origination
+
+- A new customer registers in Adobe Commerce (sign up or request)
+- A new customer registers in Adobe Commerce during or after purchase (sign up and purchase)
+- An admin user registers a new customer
```

Comment:
Customers can also be created with REST and GraphQL. I think GraphQL would only be used to complete the UI actions, but REST might be used to create accounts in bulk